main/guides/js-programming/ses/ses-reference.md

```diff
@@ -0,0 +1,316 @@
+accessible objects include but are not limited to: 
+- `globalThis`
+- `[].__proto__` the array prototype, equivalent to `Array.prototype` in a pristine JavaScript environment.
+- `{}.__proto__` the `Object.prototype` 
+- `(() => {}).__proto__` the `Function.prototype`
+- `(async () => {}).__proto__` the prototype of all asynchronous functions, and has no alias 
+   in the global scope of a pristine JavaScript environment.
+- The properties of any accessible object
+
+`lockdown()` also *tames* some objects, such as:
+- Regular expressions
+  - A tamed RexExp does not have the deprecated compile method.
+- Locale methods
+  - Lockdown replaces locale methods like `String.prototype.localeCompare()` with lexical 
+    versions that do not reveal the user locale.
+- Errors
+  - A tamed error does not have a V8 stack, but the console can still see the stack.
+   
+Lockdown does not erase any powerful objects from the initial global scope. Instead, 
+Compartments give complete control over what powerful objects exist for client code.
+
+## `lockdown()` and `harden()`
+
+`lockdown()` and `harden()` essentially do the same thing; freeze objects so their 
+properties cannot be changed. You can only interact with frozen objects through 
+their methods. Their differences are what objects you use them on, and when you use them.
+
+`lockdown()` **must** be called first. It hardens JavaScript's built-in *primordials* 
+(implicitly shared global objects) and enables `harden()`. Calling `harden()` 
+before `lockdown()` executes throws an error.
+
+`lockdown()` works on objects created by the JavaScript language itself as part of 
+its definition. Use `harden()` to freeze objects created by your JavaScript code 
+after `lockdown()`was called. 
+
+## `lockdown` Options
+
+### Default `safe` settings
+
+All four of these safety-relevant options default to `safe` if omitted 
+from a call to `lockdown()`. Their other possible value is `unsafe`.
+- `regExpTaming`
+- `localeTaming`
+- `consoleTaming`
+- `errorTaming`
+
+The tradeoff is safety vs compatibility with existing code. However, much legacy
+JavaScript code does run under SES, even if both not written to do so and with all
+the options set to `safe`. Only consider an `'unsafe'` value if you both need it 
+and can evaluate its risks.
+
+### Options quick reference
+
+This section provides a quick usage reference for `lockdown()`'s options, their possible
+values, and their usage. Each is described in more detail in their individual sections
+below.
+
+<table>
+  <tbody>
+  <tr>
+    <td><center><b>Option</b></center></td>
+    <td><center><b>Values</b></center></td>
+    <td><center><b>Functionality</b></center></td>
+  </tr>
+  <tr>
+    <td><code>regExpTaming</code></td>
+    <td><code>'safe'</code> (default) or <code>'unsafe'</code></td>
+    <td><code>'safe'</code> disables all <code>RegExp.*</code> methods,<br>
+        <code>'unsafe'</code> disables all but <code>RegExp.prototype.compile()</td>
```

Comment:
This page shows up as blank because of the unclosed html tags:

```suggestion
        <code>'unsafe'</code> disables all but <code>RegExp.prototype.compile()</code></td>
```

I'm seeing errors in the console when I inspect the page for this and `lockdown.md`, like:

<img width="1246" alt="Screen Shot 2021-03-19 at 11 42 13 AM" src="https://user-images.githubusercontent.com/2441069/111828732-f6b6f200-88a8-11eb-8ea9-0abeca93543a.png">
<img width="1247" alt="Screen Shot 2021-03-19 at 11 42 05 AM" src="https://user-images.githubusercontent.com/2441069/111828740-f880b580-88a8-11eb-90e2-38d21c8e9625.png">
<img width="1252" alt="Screen Shot 2021-03-19 at 11 41 54 AM" src="https://user-images.githubusercontent.com/2441069/111828741-f9194c00-88a8-11eb-92c6-22752dbf1033.png">